deploy/kubernetes/helm/che/values.yaml

```diff
@@ -94,6 +94,7 @@ che:
 #    pluginBroker:
 #      waitTimeoutMin: "3"
 #    pluginRegistryUrl: "https://che-plugin-registry.openshift.io/v3"
+  disableProbes: false
```

Comment:
Please consider another more flexible configuration model, like:
```yaml
che:
  probes:
    #it may be useful to disable probes if you going to debug your Che Server
    enabled: true
    readinessProbe:
       initialDelaySeconds: 120
       timeoutSeconds: 10
       ...
    livenessProbe:
       initialDelaySeconds: 15
       timeoutSeconds: 60      
```

Reply:
I don't see an immediate need for such complexity, given this was not possible at all so far and nobody complained (at least not loudly). Let's merge what we have now in this PR and iterate on it in the way you suggest once we identify the need.